### DISCUSSION TITLE
How to disable the mini-calendar

### DISCUSSION THREAD

**Discussion by Bolb389** [YU000jp/logseq-plugin-show-weekday-and-week-number#123] [Q&A]
Hello,
I tried your interesting plugin, and I have a question : is it possible to disable the mini-calendar ?
I'm mostly interested in the weekday and week number next to the journal title, and I would like to disable the mini-calendar..
Have a nice day :-)

**Comment by YU000jp** [comment]
In the plugin settings, turn off Journal Boundaries.
